pkg/ocm/storage/outcoming/ocm.go

```diff
@@ -266,22 +267,27 @@ func (d *driver) unwrappedOpFromShareCreator(ctx context.Context, share *ocmv1be
 }
 
 func (d *driver) GetMD(ctx context.Context, ref *provider.Reference, _ []string) (*provider.ResourceInfo, error) {
+	log := appctx.GetLogger(ctx)
 	share, rel, err := d.shareAndRelativePathFromRef(ctx, ref)
 	if err != nil {
+		log.Error().Err(err).Msg("Error from shareAndRelativePathFromRef")
 		return nil, err
 	}
 
 	var info *provider.ResourceInfo
 	if err := d.unwrappedOpFromShareCreator(ctx, share, rel, func(ctx context.Context, newRef *provider.Reference) error {
 		info, err = d.stat(ctx, newRef)
 		if err != nil {
+			log.Error().Err(err).Msg("Error from stat")
 			return err
 		}
 		return d.augmentResourceInfo(ctx, info, share)
 	}); err != nil {
+		log.Error().Err(err).Msg("Error from unwrappedOpFromShareCreator")
 		return nil, err
 	}
 
+	log.Debug().Interface("resinfo", info).Msg("GetMD returns")
```

Comment:
I think it's fine to add this log, but maybe to the caller and with level set to Trace